app/controllers/answers_controller.rb

```diff
@@ -9,7 +9,7 @@ def current_ability
 
   def new
     question = Question.find_by_id(params[:question_id])
-    @answer = Answer.new(value: true, question: question)
+    @answer = Answer.new(value: true, question:)
     authorize! :new, @answer
```

Comment:
What rule is it that replaces `@answer = Answer.new(value: true, question: question)`
by `@answer = Answer.new(value: true, question:)`? I did not even know that this was possible. I also think that this is bad for the readability of the code.

Reply:
Oh, good point. This is due to [`EnforcedShorthandSyntax`](https://docs.rubocop.org/rubocop/1.57/cops_style.html#enforcedshorthandsyntax-never). We can set it to `never` in order to enforce that `question: question` is used instead of `question:`.